server/constants/error.ts

```diff
@@ -1,5 +1,7 @@
 export enum ApiErrorCode {
   InvalidUrl = 'INVALID_URL',
   InvalidCredentials = 'INVALID_CREDENTIALS',
+  InvalidAuthToken = 'INVALID_AUTH_TOKEN',
   NotAdmin = 'NOT_ADMIN',
+  Generic = 'GENERIC',
```

Comment:
Just a little comment: why generic? Something like "Unknown" would be more meaningful in my opinion

Reply:
Okay